_dashboards/visualize/viz-index.md

```diff
@@ -0,0 +1,114 @@
+
+## Bar charts
+
+<img src="{{site.url}}{{site.baseurl}}/images/bar-chart-1.png" alt="Example vertical bar chart in OpenSearch Dashboards" width="60%" height="30%">
+
+<img src="{{site.url}}{{site.baseurl}}/images/bar-horizontal-1.png" alt="Example horizontal bar chart in OpenSearch Dashboards" width="60%" height="30%">
+
+Bar charts, vertical or horizontal, compare categorical data and demonstrate changes of a variable over a period of time. Learn how to create and use [Bar charts]({{site.url}}{{site.baseurl}}/dashboards/visualize/bar-charts) in Dashboards.
+
+## Data tables
+
+<img src="{{site.url}}{{site.baseurl}}/images/data-table-1.png" alt="Example data table in OpenSearch Dashboards" width="75%" height="35%">
+
+Data tables, or tables, show your raw data in tabular form. Learn how to create and use [Data tables]({{site.url}}{{site.baseurl}}/dashboards/visualize/tables) in Dashboards.
+
+## Gantt chart
+
+<img src="{{site.url}}{{site.baseurl}}/images/gantt-chart.png" alt="Example Gantt chart in OpenSearch Dashboards" width="75%" height="35%">
+
+Gantt charts show the start, end, and duration of unique events in a sequence. Gantt charts are useful in trace analytics, telemetry, and anomaly detection use cases, where you want to understand interactions and dependencies between various events in a schedule. Learn how to create and use [Gantt charts]({{site.url}}{{site.baseurl}}/dashboards/visualize/gantt/) in Dashboards.
+
+## Gauge charts
+
+<img src="{{site.url}}{{site.baseurl}}/images/gauge-chart-1.png" alt="Example gauge chart in OpenSearch Dashboards" width="60%" height="30%">
+
+A gauge chart displays how much there is of the thing you are measuring. In a gauge chart, this measurement can exist alone or in relation to another measurement, such as comparing actual sales to the sales goal. Learn how to create and use [Gauge charts]({{site.url}}{{site.baseurl}}/dashboards/visualize/gauge) in Dashboards.
+
+## Heat maps
+
+<img src="{{site.url}}{{site.baseurl}}/images/heatmap-1.png" alt="Example heat map in OpenSearch Dashboards" width="60%" height="30%">
+
+A heat map is a view of a histogram (a graphical representation of the distribution of numerical data) over time. Instead of using bar height as a representation of frequency, as with a histogram, heat maps use cells, coloring a cell proportional to the number of values. Learn how to create and use [Heat maps]({{site.url}}{{site.baseurl}}/dashboards/visualize/heat-maps) in Dashboards.
+## Line graphs
+
+<img src="{{site.url}}{{site.baseurl}}/images/line-graph-1.png" alt="Example line graph in OpenSearch Dashboards" width="60%" height="30%">
+
+Line charts compare changes in measure values over period of time, such as gross sales by month or gross sales and net sales by month. Learn how to create and use [Line charts]({{site.url}}{{site.baseurl}}/dashboards/visualize/line-charts) in Dashboards.
```

Comment:
Should this be something like "Reference [Line charts]({{site.url}}{{site.baseurl}}/dashboards/visualize/line-charts) to learn how to create and use Line charts in Dashboards." ?